src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToRunCrossgen.cs

```diff
@@ -121,13 +125,15 @@ public void It_creates_readytorun_symbols_when_switch_is_used(string targetFrame
                 });
             }
 
-            DoesImageHaveR2RInfo(mainProjectDll).ToString().Should().Be(true.ToString());
-            DoesImageHaveR2RInfo(classLibDll).ToString().Should().Be(true.ToString());
+            DoesImageHaveR2RInfo(mainProjectDll).Should().BeTrue();
+            DoesImageHaveR2RInfo(classLibDll).Should().BeTrue();
+
+            publishDirectory.Should().HaveFile("System.Private.CoreLib.dll"); // self-contained
```

Comment:
Nit: it would be nice to duplicate this test for the framework dependent scenario, to validate the behavior of the `_CreateR2RSymbols` target

Reply:
Fixed.